docs/advanced/kotlin-codegen.md

```diff
@@ -0,0 +1,85 @@
+# About
```

Comment:
For better PR reviewing in the future, I recommend putting each sentence on a new line. The rendering will be the same, but you can more specifically put a comment on a line.

Reply:
Done